src/psycopmlutils/loaders/raw/load_diagnoses.py

```diff
@@ -254,3 +254,370 @@ def sleep_problems_unspecified(n: Optional[int] = None) -> pd.DataFrame:
             wildcard_icd_code=False,
             n=n,
         )
+
+    @data_loaders.register("all f0-disorders")
+    def all_f0_disorders(n: Optional[int] = None) -> pd.DataFrame:
```

Comment:
Would probably rename this to just `"f0 disorders" ` etc. and the function names to `f0_disorders()`

Reply:
Corrected in new commit.